orchestrator-salt/src/main/resources/salt-common/salt/monitoring/textfiles.sls

```diff
@@ -0,0 +1,41 @@
+{%- from 'monitoring/settings.sls' import monitoring with context %}
+
+{%- if monitoring.enabled and monitoring.nodeExporterExists %}
+
+/var/lib/node_exporter/files:
+  file.directory:
+    - name: /var/lib/node_exporter/files
+    - user: "root"
+    - group: "root"
+    - mode: 750
+    - failhard: True
+    - makedirs: True
+
+/var/lib/node_exporter/scripts:
+  file.directory:
+    - name: /var/lib/node_exporter/scripts
+    - user: "root"
+    - group: "root"
+    - mode: 750
+    - failhard: True
+    - makedirs: True
+
+/etc/cron.d/salt_key_check:
+  file.managed:
+    - name: /etc/cron.d/salt_key_check
```

Comment:
is there any reason cron job is created via file instead of cron state?
https://docs.saltproject.io/en/latest/ref/states/all/salt.states.cron.html